app/Filament/Pages/Settings.php

```diff
@@ -159,23 +159,7 @@ private function generalSettings(): array
                         ->label('Set to Cloudflare IPs')
                         ->icon('tabler-brand-cloudflare')
                         ->authorize(fn () => auth()->user()->can('update settings'))
-                        ->action(fn (Set $set) => $set('TRUSTED_PROXIES', [
-                            '173.245.48.0/20',
-                            '103.21.244.0/22',
-                            '103.22.200.0/22',
-                            '103.31.4.0/22',
-                            '141.101.64.0/18',
-                            '108.162.192.0/18',
-                            '190.93.240.0/20',
-                            '188.114.96.0/20',
-                            '197.234.240.0/22',
-                            '198.41.128.0/17',
-                            '162.158.0.0/15',
-                            '104.16.0.0/13',
-                            '104.24.0.0/14',
-                            '172.64.0.0/13',
-                            '131.0.72.0/22',
-                        ])),
+                        ->action(fn (Set $set) => $set('TRUSTED_PROXIES', config('trustedproxy.cloudflare'))),
```

Comment:
Pull from the official source instead: https://www.cloudflare.com/ips-v4/ and https://www.cloudflare.com/ips-v6/

Reply:
I'll pull from https://api.cloudflare.com/client/v4/ips so i get both in a single query

>See that tomorrow